spec/features/users_can_edit_user_info_spec.rb

```diff
@@ -3,24 +3,24 @@
 feature "User can edit User info" do
   before do
     user = User.create(first_name: "Jane",
-                       last_name:  "Doe",
-                       username:   "Jane's Shop",
-                       email:      "jane@doe.com",
-                       password:   "password")
+      last_name:  "Doe",
+      username:   "Jane's Shop",
+      email:      "jane@doe.com",
+      password:   "password")
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.